packages/python/plotly/plotly/tests/test_core/test_px/test_trendline.py

```diff
@@ -0,0 +1,12 @@
+import plotly.express as px
+import numpy as np
+
+
+def test_trendline_nan_values():
```

Comment:
can we expand this test to cover both trendline types, and assert that the lengths of x and y are the same? I think that was not true before but should always be.

Reply:
Done ! Indeed x and y could have different lengths before this PR.

Reply:
awesome, thanks!